src/coreclr/vm/virtualcallstub.cpp

```diff
@@ -954,6 +1087,30 @@ BOOL VirtualCallStubManager::TraceManager(Thread *thread,
 
 #ifndef DACCESS_COMPILE
 
+DispatchToken VirtualCallStubManager::GetTokenFromOwnerAndSlot(TypeHandle ownerType, uint32_t slot)
+{
+    CONTRACTL
+    {
+        THROWS;
+        GC_TRIGGERS;
+        MODE_ANY;
+        INJECT_FAULT(COMPlusThrowOM(););
+    }
+    CONTRACTL_END
+
+    MethodTable * pMT = ownerType.GetMethodTable();
+    pMT->GetRestoredSlot(slot);
```

Comment:
`pMT` can be `NULL` if `ownerType` is a `TypeDesc` that isn't a VT or ptr, is that possible here? I'm guessing it's not but just wanted to check

Reply:
Yeah, can't happen. Those cases are for TypeHandle's that can't have methods.